pkgs/games/steam/fhsenv.nix

```diff
@@ -157,7 +157,7 @@ in buildFHSUserEnv rec {
     xorg.libXdamage
     xorg.libxshmfence
     xorg.libXxf86vm
-    llvm_11.lib
+    llvmPackages_latest.llvm.lib
```

Comment:
If

> LLVM must match version required by Mesa

is the justification, please add it here as a comment, so that we know in the future.